packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoHamburgerMenu.tsx

```diff
@@ -1,41 +1,61 @@
 import React, { useState } from 'react';
 
 import { signOut } from 'next-auth/react';
-import { useTranslations } from 'next-intl';
 
 import { EuiButtonIcon, EuiContextMenu, EuiPopover } from '@elastic/eui';
 import type { EuiContextMenuPanelDescriptor } from '@elastic/eui';
 
-import { useOrchestratorTheme } from '@/hooks';
+import { useGetOrchestratorConfig, useOrchestratorTheme } from '@/hooks';
 import { WfoLogoutIcon } from '@/icons';
+import { WfoQuestionCircle } from '@/icons/WfoQuestionCircle';
 
 export const WfoHamburgerMenu = ({}) => {
     const [isPopoverOpen, setPopoverIsOpen] = useState(false);
     const { theme, isDarkThemeActive } = useOrchestratorTheme();
-    const t = useTranslations('main');
-
+    const { enableSupportMenuItem, supportMenuItemUrl } =
+        useGetOrchestratorConfig();
     const closePopover = () => {
         setPopoverIsOpen(false);
     };
+    const handleOpenSupport = async (): Promise<undefined> => {
+        window.open(supportMenuItemUrl, '_blank');
+    };
+
+    const panelItems = [
+        {
+            name: 'Logout',
+            icon: (
+                <WfoLogoutIcon
+                    color={
+                        isDarkThemeActive
+                            ? theme.colors.ghost
+                            : theme.colors.ink
+                    }
+                />
+            ),
+            onClick: () => signOut(),
+        },
+    ];
+
+    enableSupportMenuItem &&
+        panelItems.splice(0, 0, {
```

Comment:
Try to avoid mutating. Can use toOptionalArrayEntry from the utils: `packages/orchestrator-ui-components/src/utils/optionalArray.ts`.

`const panelItems = [ { logoutItem }, ...toOptionalArray(....) ] `